src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public static boolean validate(String password, String repeatPassword)
```

Comment:
Why do we make this method static?

Reply:
As I understood it is an advanced method that is not specialized for a particular class or a person. And to short our method registerUser in class in class UserService.  =)

Reply:
To short our code without calling instance of class in method registerUser in class in class UserService. =)

Reply:
this should not be the main goal when writing code) 
you need to follow certain architectural principles